driver/csiplugin/controllerserver.go

```diff
@@ -161,7 +163,7 @@ func (cs *ScaleControllerServer) GenerateVolId(scVol *scaleVolume) (string, erro
 	uid, err := scVol.PrimaryConnector.GetFsUid(scVol.LocalFS)
 	glog.Infof("GetFsUID error [%v] uid [%v]", err, uid)
```

Comment:
This message is getting printed as " GetFsUID error [<nil>] uid [C0A87A54:5EE77349]" and seems confusing whether it is a error message or information while error is nil.